### PR TITLE
Migration tool enhancement

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: app/components/shared/HighlightedStripField.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  value,
+  icon,
+  link,
+  withCopy,
+  whiteBackground,
+  dataTestId,
+}) => {
+  return (
+    <Container>
+      <Row
```
Comment (trival) [inline app/components/shared/HighlightedStripField.tsx]: This looks pretty similar to our InfoList component, but always good to have a backup : )

File: app/components/shared/buttons/Button.tsx [hunk 2]
```diff
@@ -103,9 +106,9 @@ Button.defaultProps = {
   width: ButtonWidth.NORMAL,
 };
 
-const ButtonIcon: React.SFC<IButtonIcon> = ({ onClick, className, ...props }) => (
+const ButtonIcon: React.SFC<IButtonIcon> = ({ onClick, className, width, height, ...props }) => (
   <button className={cn(styles.buttonIcon, className)} onClick={onClick}>
-    <InlineIcon {...props} width="20px" height="20px" />
+    <InlineIcon {...props} width={width || "20px"} height={height || "20px"} />
```
Comment (trival) [inline app/components/shared/buttons/Button.tsx]: We should get rid of with and height completely, see https://github.com/Neufund/platform-frontend/pull/1699

File: app/components/modals/icbm-wallet-balance-modal/IcbmWalletBalanceModal.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    migrationInputData: "0x1234",
+    gasLimit: "40000",
+    value: "0",
+  },
+];
+
+storiesOf("ICBMWalletModalComponant", module)
+  .add("Missing Verifications", () => (
+    <IcbmWalletBalanceComponent
+      isOpen={true}
```
[the remark this answers is not in the document]
Comment (Moejoe90) [reply app/components/modals/icbm-wallet-balance-modal/IcbmWalletBalanceModal.stories.tsx]: This is already the case. Btw
If you check storybook all the states are represented including the Balance modal and Migrate.
I came with the idea of choice between showing each modal separately or as a full story. 

I think it's better as a full story like this

File: app/components/modals/icbm-wallet-balance-modal/IcbmWalletBalanceModal.tsx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+> {
+  state = {
+    migrationStep: this.props.currentMigrationStep || 1,
+    migrationData: this.props.walletMigrationData,
+    success: this.props.success || false,
+    isMigrating: this.props.isMigrating || false,
+  };
+
+  private startMigration = () => {
+    this.setState({ isMigrating: true });
```
[the remark this answers is not in the document]
Comment (Moejoe90) [reply app/components/modals/icbm-wallet-balance-modal/IcbmWalletBalanceModal.tsx]: Very valid point.
This pr started as a refactor and for some reason i always kept the idea of having state.

You have a valid point

**Pull request merged**